psi4/driver/json_wrapper.py

```diff
@@ -52,11 +55,126 @@
     "dipole", "quadrupole", "mulliken_charges", "lowdin_charges", "wiberg_lowdin_indices", "mayer_indices"
 }
 
+## QCSchema translation blocks
+
+_qcschema_translation = {
+
+    # Generics
+    "generics": {
+        "return_energy": {"variables": "CURRENT ENERGY"},
+        "nuclear_repulsion_energy": {"variables": "NUCLEAR REPULSION ENERGY"},
+    },
+
+    # Properties
+    "properties": {
+        "mulliken_charges": {"variables": "MULLIKEN_CHARGES", "skip_null": True},
+        "lowdin_charges": {"variables": "LOWDIN_CHARGES", "skip_null": True},
+        "wiberg_lowdin_indices": {"variables": "WIBERG_LOWDIN_INDICES", "skip_null": True},
+        "mayer_indices": {"variables": "MAYER_INDICES", "skip_null": True},
+    },
+
+    # SCF variables
+    "scf": {
+        "scf_one_electron_energy": {"variables": "ONE-ELECTRON ENERGY"},
+        "scf_two_electron_energy": {"variables": "TWO-ELECTRON ENERGY"},
+        "scf_dipole_moment": {"variables": ["SCF DIPOLE X", "SCF DIPOLE Y", "SCF DIPOLE Z"]},
+        "scf_iterations": {"variables": "SCF ITERATIONS", "cast": int},
+        "scf_total_energy": {"variables": "SCF TOTAL ENERGY"},
+        "scf_vv10_energy": {"variables": "DFT VV10 ENERGY", "skip_zero": True},
+        "scf_xc_energy": {"variables": "DFT XC ENERGY", "skip_zero": True},
+        "scf_dispersion_correction_energy": {"variables": "DISPERSION CORRECTION ENERGY", "skip_zero": True},
+
+        # SCF Properties (experimental)
+        # "scf_quadrupole_moment": {"variables": ["SCF QUADRUPOLE " + x for x in ["XX", "XY", "XZ", "YY", "YZ", "ZZ"]], "skip_null": True},
+        # "scf_mulliken_charges": {"variables": "MULLIKEN_CHARGES", "skip_null": True},
+        # "scf_lowdin_charges": {"variables": "LOWDIN_CHARGES", "skip_null": True},
+        # "scf_wiberg_lowdin_indices": {"variables": "WIBERG_LOWDIN_INDICES", "skip_null": True},
+        # "scf_mayer_indices": {"variables": "MAYER_INDICES", "skip_null": True},
+    },
+
+    # MP2 variables
+    "mp2": {
+        "mp2_same_spin_correlation_energy": {"variables": "MP2 SAME-SPIN CORRELATION ENERGY"},
+        "mp2_opposite_spin_correlation_energy": {"variables": "MP2 OPPOSITE-SPIN CORRELATION ENERGY"},
+        "mp2_singles_energy": {"variables": "NYI", "default": 0.0},
+        "mp2_doubles_energy": {"variables": "MP2 CORRELATION ENERGY"},
+        "mp2_total_correlation_energy": {"variables": "MP2 CORRELATION ENERGY"},
+        "mp2_total_energy": {"variables": "MP2 TOTAL ENERGY"},
```

Comment:
This mp2 section currently runs afoul of OCC, as I noticed when running CBS through json b/c some variables not set. I think this'll be fixed by another hunk of #1381, so just a heads-up.